src/OrchardCore/OrchardCore.DisplayManagement/Shapes/AlternatesCollection.cs

```diff
@@ -26,7 +26,7 @@ public AlternatesCollection(params string[] alternates)
 
         public string this[int index] => _collection[index];
 
-        public string Last => _collection[_collection.Count - 1];
+        public string Last => _collection.Count >0 ? _collection[^1]: "";
```

Comment:
If you use our source code with our `.editorconfig` file and under Visual Studio, try to `ctrl K+D`, you will see that it adds a space after the `>`.

I learnt something today about c# 8.0, thanks ;)